index.js

```diff
@@ -41,7 +41,7 @@ export async function generateNotes(pluginConfig, context) {
   protocol = protocol && /http[^s]/.test(protocol) ? "http" : "https";
   const [, owner, repository] = /^\/(?<owner>[^/]+)?\/?(?<repository>.+)?$/.exec(pathname);
 
-  const { issue, commit, referenceActions, issuePrefixes } =
+  const { issue, commit, referenceActions, issuePrefixes, repositoryPathSuffix } =
```

Comment:
```suggestion
  const { issue, commit, referenceActions, issuePrefixes, repositoryPathSuffix = "" } =
```
I could also default to an empty string if you'd like and just do
```
repository:`${repository}${repositoryPathSuffix}`,
```